packages/subapp-web/src/subapp-web.js

```diff
@@ -1,6 +1,10 @@
 // xarc subapp client side lib version 1
 // load into window.xarcV1 as a global
 (function (w) {
+  if (!w._wml) {
```

Comment:
@jchip we were already initializing `w._wml` in `cdn-map.js` (https://github.com/electrode-io/electrode/blob/master/packages/subapp-web/src/cdn-map.js#L31). 

But now, since we don't load `cdn-map.js` for dev, added this here also. 

If we dont initialize this, it will throw error for `electrode-ui-config`  where we try to set config values in `window._wml`